readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://Dmytro-Yankovskyi.github.io/layout_search-bar-airbnb/)
```

Comment:
https://user-images.githubusercontent.com/38065883/175111996-7d589c2a-667f-4475-9d0c-f62d4cbc254a.mov

I see no linear-gradient on focus(design input has it)